programming/sed-article.md

```diff
@@ -351,6 +333,7 @@ For example, `/abc|def|ghi/` matches text containing
 "abc", "def", or "ghi".
 The patterns before and after the `|` can be any kind
 of regular expression pattern, not just literal text.
+The order of precedence in regular expressions is grouping, concatenation, and alternation.
```

Comment:
What do you mean by "concatenation"?
Is it just characters that must follow other characters?

Reply:
Yes.  ab|cd matches ab or cd, not a followed by b or c, followed by d.

Reply:
Is "alternation" the correct term to use here?
The dictionary definition is "the repeated occurrence of two things in turn."
That doesn't seem to fit. Maybe I should use "alternatives" instead.

Reply:
I've seen it as "alternation" in books on formal languages.  Alternative would probably work as well.